orchestrator-salt/src/main/resources/salt/salt/ambari/scripts/setup-ldap.sh

```diff
@@ -8,7 +8,7 @@ yes n | ambari-server setup-ldap \
   --ldap-secondary-url="{{ ldap.serverHost }}:{{ ldap.serverPort }}" \
   --ldap-ssl="{{ ambari.secure_ldap }}" \
   --ldap-user-class="{{ ldap.userObjectClass }}" \
-  --ldap-user-attr="{{ ldap.groupNameAttribute }}" \
+  --ldap-user-attr="{{ ldap.userNameAttribute }}" \
```

Comment:
@keyki I am not sure why it is changed! @jtstorck do you know that?

Reply:
@doktoric The group name attribute was being used for the ldap-user-attr param, when it should've been the username attribute.  @keyki flagged that when I had some Ambari ldap setup issues when I started testing the LDAP integration.